apps/application-system/form/infra/application-system-form.ts

```diff
@@ -1,25 +1,12 @@
 import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
 
-export const serviceSetup = (services: {
-  api: ServiceBuilder<'api'>
-}): ServiceBuilder<'application-system-form'> =>
+export const serviceSetup = (): ServiceBuilder<'application-system-form'> =>
   service('application-system-form')
     .namespace('application-system')
     .liveness('/liveness')
     .readiness('/readiness')
     .env({
       BASEPATH: '/umsoknir',
-      SI_PUBLIC_GRAPHQL_PATH: {
```

Comment:
This has not been fully refactored out yet. There are still 2 applications that are still using the GQL 🤕.  [PSign](https://github.com/island-is/island.is/blob/main/libs/application/template-api-modules/src/lib/modules/templates/p-sign-submission/p-sign-submission.service.ts#L98) and [General Petition](https://github.com/island-is/island.is/blob/main/libs/application/template-api-modules/src/lib/modules/templates/general-petition/general-petition.service.ts#L58).

Reply:
Sry this is the form that should not affect the template api.